script/refresh

```diff
@@ -4,9 +4,19 @@ set -euo pipefail
 
 ROOT=$(cd $(dirname $0)/..; pwd)
 
-DOCKER_MACHINE_NAME=${DOCKER_MACHINE_NAME:-dev}
+# Are we using Docker for Mac?
+OSX=$(docker info --format "{{.OperatingSystem}}" | grep -q "Docker for Mac")
+if [[ ${OSX} -eq 0 ]]; then
+    DECONST_IP="127.0.0.1"
+else
+    DOCKER_MACHINE_NAME=${DOCKER_MACHINE_NAME:-dev};
+    DECONST_IP=$(docker-machine ip ${DOCKER_MACHINE_NAME})
+fi
 
-DECONST=$(docker-machine ip ${DOCKER_MACHINE_NAME})
+if ! [[ -x "$(command -v grunt)" ]]; then
+    echo "Error: grunt command not found. Please install it."
```

Comment:
Minor note: I'd reword this error to be something like this to be a bit closer to the common shell error message. Might help folks who aren't too familiar with the terminal:
```bash
echo "Error: grunt: command not found. Install 'grunt' and retry."
```